invoices_app/__init__.py

```diff
@@ -65,9 +61,30 @@
 
 THANK_YOU = _('Thank you')
 EVENTBRITE_INVOICE_COMMENTED = _('Eventbrite Invoice {} commented')
-NEW_COMMENT_EMAIL_TEXT = _('You have a new comment on Invoice # {}. Please check your invoice. COMMENT: {}')
-INVOICE_CHANGE_STATUS_TEXT_EMAIL = _('Invoice {} changed status to {}')
+NEW_COMMENT_EMAIL_TEXT = _('An Eventbrite administrator left a new comment on your Invoice #{}. Please check your '
+                           'invoice on BriteSu. COMMENT: {}')
+INVOICE_CHANGE_STATUS_TEXT_EMAIL = _('Invoice #{} for Eventbrite is now {}')
 EVENTBRITE_INVOICE_EDITED = _('Eventbrite Invoice Edited')
 INVOICE_EDIT_INVOICE_UPPER_TEXT = _('Your Invoice # {} was edited by an administrator. Please check your invoice')
+INVOICE_STATUS_APPROVED_UPPER = _('Your invoice #{} for Eventbrite has been approved')
+INVOICE_STATUS_APPROVED_EMAIL = _('Your invoice #{} for Eventbrite has been approved. '
+                                  'We’ll let you know once it is paid.')
+
+INVOICE_STATUS_CHANGES_REQUEST_UPPER = _('Your Invoice #{} for Eventbrite has required changes')
+INVOICE_STATUS_CHANGES_REQUEST_EMAIL = _('An Eventbrite administrator has required some changes on invoice #{} '
+                                         'Please check your invoice on BriteSu and apply the required changes. '
+                                         'COMMENT: {}')
+
+INVOICE_STATUS_IN_PROGRESS_EMAIL = _('Your invoice #{} is being processed by Eventbrite. We’ll let you know about its '
+                                     'progress soon.')
+
+INVOICE_STATUS_REJECTED_UPPER = _('Your invoice #{} for Eventbrite has been rejected')
+INVOICE_STATUS_REJECTED_EMAIL = _('Your invoice #{} for Eventbrite has been rejected. '
+                                  'Please login into BriteSu to see any additional information.')
+
+INVOICE_STATUS_PAID_UPPER = _('Your invoice #{} for Eventbrite has been paid')
+INVOICE_STATUS_PAID_EMAIL = _('Your invoice #{} for Eventbrite has been paid. '
+                              'Please login into BriteSu to see any additional information.')
 
-DISCLAIMER = _('"Please do not reply to this email. If you have any questions, please login into Britesu and make a comment in the section related to your question."'),
\ No newline at end of file
+DISCLAIMER = _('"Please do not reply to this email. If you have any questions, please login into Britesu and make a '
```

Comment:
I think, based in previous occurrences, that you should write BriteSu with an S in capital manner